src/components/nav/MusicBar2.vue

```diff
@@ -439,6 +479,26 @@ export default {
             });
         },
         songBuffering() {},
+        mathFloorTime() {
+            return Math.floor(this.songProgressTimestamp);
+        },
+        timeFormat() {
+            const Stringtime = formatDuration(Math.floor(this.songProgressTimestamp * 1000));
+            return Stringtime;
+        },
+        mouseOver() {
+            // dont turn to !this.showNavbar and !this.MouseEnter cause it trigger multiple times in their child element too
+            this.coolDown = 0;
```

Comment:
logic for cooldown doesn't really make sense, since it's not actually encoding a cooldown value. it gets set to `thisTime` later, which is a marker-like behavior.